README.md

```diff
@@ -311,8 +311,7 @@ described in [RFC 8032]
  * [link-header](https://github.com/spider-gazelle/link-header) - HTTP Link Header Parser
  * [ntlm](https://github.com/spider-gazelle/ntlm) - NTLM authentication
  * [proxy-fetcher.cr](https://github.com/nbulaj/proxy-fetcher.cr) - Proxy lists fetching & validating library
- * [sse.cr](https://github.com/y2k2mt/sse.cr) - [Server-Sent Events](https://www.w3.org/TR/2009/WD-eventsource-20090421) client
-
+ * [sse.cr](https://github.com/y2k2mt/sse.cr) - [Server-Sent Events](https://github.com/y2k2mt/sse.cr) client
```

Comment:
```suggestion
 * [sse.cr](https://github.com/y2k2mt/sse.cr) - [Server-Sent Events](https://github.com/y2k2mt/sse.cr) client
 
```

Reply:
I don't' think the 2nd link was meant to link to the shard itself also. Probably would make more sense to link to like https://html.spec.whatwg.org/multipage/server-sent-events.html

Reply:
```suggestion
 * [sse.cr](https://github.com/y2k2mt/sse.cr) - [Server-Sent Events](https://html.spec.whatwg.org/multipage/server-sent-events.html) client
```